src/praisonai-agents/praisonaiagents/messaging/protocols.py

```
@@ -0,0 +1,129 @@
+        Args:
+            recipient: Address of the receiving agent.
+            body: Message payload.
+            sender: Address of the sending agent.
+            correlation_id: Optional id to correlate a reply.
+
+        Returns:
+            The id of the delivered message.
+        """
+        ...
+
+    def receive(self, agent_id: str, *, max: int = 50) -> list[AgentMessage]:
```

Ruff (0.15.21)

[error] Function argument `max` is shadowing a Python builtin

(A002)